libs/prisma-models/src/record.rs

```diff
@@ -293,6 +293,22 @@ impl Record {
         Ok(RecordIdentifier { pairs })
     }
 
+    pub fn identifying_values(&self, field_names: &[String], id: &ModelIdentifier) -> crate::Result<Vec<&PrismaValue>> {
```

Comment:
This is similar to the `.identifiers` method. Is the only reason we need this here the references instead of the clones? Did that make a large difference?